Anastasia Dubilo/src/components/Post.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+class Post extends React.Component {
+    render() {
+        return (<div className="post">
+                <h2 className="postTitle">{this.props.children.title}</h2>
+                <div className="postBody">{textToHtml(this.props.children.postBody)}</div>
```

Comment:
```suggestion
                <div className="postBody">{this.textToHtml(this.props.children.postBody)}</div>
```
при условии, что `textToHtml()` - метод